README.md

```diff
@@ -22,26 +22,31 @@ This gradle plugin can be used with any Android project in Java or Kotlin.
 
 # 1 - Install the plugin
 
-Get the latest version of the plugin from [releases](https://github.com/klaxit/hidden-secrets-gradle-plugin/releases).
-Copy `HiddenSecretsPlugin-X.Y.Z.jar` to your Android project in `/app/libs/` folder.
+## Using the plugins DSL
+In your app level `build.gradle`:
+
+```gradle
+plugins {
+    id "com.klaxit.hiddensecrets" version "X.Y.Z"
+}
```

Comment:
I would recommend this:

```suggestion

// build.gradle
plugins {
    id "com.klaxit.hiddensecrets" version "X.Y.Z"
}

// build.gradle.kts
plugins {
    id("com.klaxit.hiddensecrets") version "X.Y.Z"
}
```

Reply:
@Mazorius Thanks for your recommandation but we would like to keep the readme file as clear as possible. I think people using kts gradle files will know that they need to add parenthesis ?

Reply:
Maybe we do not need to add anything of this 🤔 

As soon as you have published the plugin to plugins.gradle.org you get this: https://plugins.gradle.org/plugin/com.palantir.metric-schema

Maybe a link inside your readme is then enough information?

Reply:
Yes I added a link to https://plugins.gradle.org/plugin/com.klaxit.hiddensecrets for all details.